### PR TITLE
Bump version to v6.0.3

### PR DESCRIPTION
**Pull request by andreas-bok-sociomantic**
New ABI version 13.0.3

Use v6.0.3 in the description for how to link the
library

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -100,13 +100,13 @@ refer to `compiling.rst`_ or build the documentation, cf. below.
...
-version ``v6.0.2`` linking your app against libdrizzle-redux requires the flag
+version ``v6.0.3`` linking your app against libdrizzle-redux requires the flag
 ``-ldrizzle-redux``::
 
     g++ app.c -oapp -ldrizzle-redux -lssl -lcrypto -pthread
 
-From version ``v6.0.2`` and later the API level of the library is appended to
-the installed library name. Thus, linking against ``libdrizzle-redux v6.0.2``
+From version ``v6.0.3`` and later the API level of the library is appended to
+the installed library name. Thus, linking against ``libdrizzle-redux v6.0.3``
```
Comment (mathias-lang-sociomantic) [inline README.rst]: No mention of the headers path ?

**Comment by andreas-bok-sociomantic** [comment]
Blocked on #174

**Comment by mathias-lang-sociomantic** [comment]
Rebase plz

**Comment by andreas-bok-sociomantic** [comment]
Rebased

**Comment by ben-palmer-sociomantic** [comment]
LGTM, should it still be blocked?

**Review by mathias-lang-sociomantic — APPROVED** [review]
LGTM

**Pull request merged**